nginx.conf

```diff
@@ -0,0 +1,10 @@
+server {
+  listen 80;
+  server_name _ default_server;
+
+  root /usr/share/nginx/html;
+
+  location / {
+    try_files $uri $uri/ /index.html;
```

Comment:
this redirects everything to the index.html so that the /abc123 links are working with the vue router.